static/scss/detail/who-should-enroll.scss

```diff
@@ -0,0 +1,93 @@
+// sass-lint:disable mixins-before-declarations
+.who-should-enroll-block {
+  width: 100%;
+  padding: 150px 0 50px;
+  background: #2c2a29 url("/static/images/who-should-enroll-bg.png") repeat-x 0 0;
+  min-height: 400px;
+  overflow: hidden;
+  position: relative;
+  margin: -50px 0 0;
+  color: white;
+  text-align: center;
+
+  @include media-breakpoint-up(lg) {
+    background-position: 0 100%;
+    text-align: left;
+  }
+
+  &:after {
+    position: absolute;
+    left: 50%;
+    top: -350px;
+    content: "";
+    background: white;
+    width: 1200px;
+    height: 400px;
+    border-radius: 50%;
+    margin: 0 0 0 -600px;
+    @include media-breakpoint-up(md) {
+      margin: 0 0 0 -1500px;
+      width: 3000px;
+    }
+  }
+
+  [class^="col-"],
+  [class*=" col-"],
+  .container {
+    @include media-breakpoint-up(lg) {
+      position: static;
+      min-height: 505px;
+    }
+  }
+
+  .img-holder {
+    padding: 40px 0 0;
+    @include media-breakpoint-up(lg) {
+      position: absolute;
+      right: 0;
+      top: 150px;
+      width: calc(50% - 40px);
+      padding: 0;
+    }
+
+    img {
+      display: block;
+      width: 100%;
+      height: auto;
+      @include media-breakpoint-up(lg) {
+        height: 505px;
+        object-fit: cover;
+      }
+    }
+  }
+
+  h1 {
+    font-size: 40px;
+    line-height: 45px;
+    color: $who-should-enroll--heading;
```

Comment:
I'm thinking we should do the following:
- Move this variable into this file (since it's only being used here), call it `$dark-bg-heading-color`, and define it at the top of the file
- Make a variable out of the block's bg color (`#2c2a29`) and call it `$darker-grey-bg`

Reply:
Done

Reply:
Done.